src/less/select/base/select.less

```diff
@@ -47,6 +47,20 @@ span.select {
     width: 100%;
 }
 
+.select--underline {
+    select {
+        border: none;
+        border-bottom: 1px solid @select-underline-border-bottom-color;
+        font-weight: bold;
+        height: 1.875rem; // a11y: has to be relative to the text size
+        padding-left: 0;
+    }
+
+    svg.icon {
+        right: 0;
```

Comment:
Should add RTL support as well for this, just need to add this `.select--underline svg.icon { right: auto; left: 0 }` to the RTL section in this file